code/modules/mob/living/simple_animal/hostile/illusion.dm

```diff
@@ -22,11 +24,15 @@
 		death()
 
 
-/mob/living/simple_animal/hostile/illusion/proc/Copy_Parent(mob/living/original, life = 50)
+/mob/living/simple_animal/hostile/illusion/proc/Copy_Parent(mob/living/original, life = 50, health = 100, damage = 0, replicate = 0 )
 	appearance = original.appearance
 	parent_mob = original
 	dir = original.dir
-	life_span = world.time+life //5 seconds
+	life_span = world.time+life
+	melee_damage_lower = damage
+	melee_damage_upper = damage
+	multiply_chance = replicate
+	faction -= "neutral"
```

Comment:
I only ask because when I was accomplishing the same thing in #9150 I ran a check to see if removing a faction was even necessary. I never tried just doing it straight up like this.